aksel.nav.no/website/components/layout/god-praksis-page/cards/GpArticleCard.tsx

```diff
@@ -0,0 +1,77 @@
+import cl from "clsx";
+import Link from "next/link";
+import { FileFillIcon, TagFillIcon } from "@navikt/aksel-icons";
+import { BodyLong, Detail, Heading } from "@navikt/ds-react";
+import { AnimatedChevron } from "@/web/AnimatedChevron";
+
+type GpArticleCardProps = {
+  children: React.ReactNode;
+  href: string;
+  innholdstype?: string;
+  undertema?: string;
+  description?: string;
+  publishedAt?: string;
+};
+
+function GpArticleCard({
+  href,
+  children,
+  innholdstype,
+  undertema,
+  description,
+  publishedAt,
+}: GpArticleCardProps) {
+  return (
+    <div className="group relative flex flex-col gap-1 rounded-lg bg-surface-default p-4 pb-3 text-text-default shadow-xsmall hover:shadow-small has-[:focus-visible]:shadow-focus md:p-5 md:pb-4">
+      <Link
+        href={`/${href}`}
+        className="peer after:absolute after:inset-0 after:z-10 after:rounded-lg focus:outline-none"
+      >
+        <Heading
+          size="small"
+          as="h3"
+          className="text-aksel-heading underline group-hover:no-underline"
+        >
+          {children}
+        </Heading>
+      </Link>
+      {publishedAt && (
+        <Detail as="time" textColor="subtle" uppercase>
+          {publishedAt}
+        </Detail>
+      )}
+
+      {description && (
+        <BodyLong className="mt-1 line-clamp-2">{description}</BodyLong>
+      )}
+      <div
+        className={cl("mt-auto flex h-fit justify-between text-deepblue-700", {
+          "pt-5": !!description,
```

Comment:
Tror ikke `!!` er nødvendig her (synes det er mer lesbart uten)
```suggestion
          "pt-5": description,
```